go.mod

```diff
@@ -1,17 +1,15 @@
 module github.com/google/fscrypt
 
-go 1.23.0
-
-toolchain go1.23.12
+go 1.24.0
```

Comment:
Note that this increases the minimum Go version to 1.24, but only versions 1.24 and 1.25 are supported, so that seems fine.